helm/charts/determined/values.yaml

```diff
@@ -38,17 +38,17 @@ db:
   # hostAddress:
 
   # Required parameters, whether you are using your own DB or a Determined DB.
-  name: determined
-  user: postgres
-  password: postgres
+  name: coreweave_determined
+  user: coreweave_postgres
+  password: coreweave_postgres
   port: 5432
 
   # Only used for Determined DB deployment. Configures the size of the PersistentVolumeClaim for the
   # Determined deployed database, as well as the CPU and memory requirements. Should be adjusted for
   # scale.
-  storageSize: 30Gi
-  cpuRequest: 2
-  memRequest: 8Gi
+  storageSize: 100Gi
```

Comment:
should this be configurable?

Reply:
Yeah I wonder that too, have no idea how big it can get?